lib/python/docs/notebooks/experiment_tracking_demo.ipynb

```diff
@@ -211,7 +202,7 @@
    "source": [
     "# Setting local tracking URI and experiment name\n",
     "mlflow.set_tracking_uri(uri=\"http://127.0.0.1:5000\")\n",
-    "mlflow.set_experiment(\"Demonstator\")\n",
+    "mlflow.set_experiment(\"hgfc\")\n",
```

Comment:
Maybe change this back to something less abstract?

Reply:
Done